easybuild/easyblocks/w/wrf.py

```diff
@@ -81,6 +81,7 @@ def extra_options():
             'rewriteopts': [True, "Replace -O3 with CFLAGS/FFLAGS", CUSTOM],
             'runtest': [True, "Build and run WRF tests", CUSTOM],
             'withvortex': [False, "Build with Vortex support", CUSTOM],
+            'withhydro': [True, "Build with Hydro support", CUSTOM],
```

Comment:
```suggestion
            'withhydro': [False, "Build with Hydro support", CUSTOM],
```

Then the easyconfig should have `withhydro = True` in it. Note that the pipeline on the easyconfig repo will then fail until this PR has been merged.